ui/src/containers/AppsTableContainer.js

```diff
@@ -30,7 +30,13 @@ export class AppsTableContainer extends React.Component {
   getTable () {
     return (
```

Comment:
You could use the spread operator to extract the props here, so that you don't have to do `this.props.x.x...`:

```jsx
const { columns, apps, sortBy } = this.props;
```

Reply:
good point, we can address in another PR

Reply:
Resolving this as unrelated to code changes in this PR. We can use spread operator to extract props in AppDetailsContainer, but to extract 2 props the juice doesn't seem worth the squeeze.